bucket/geogebra5.json

```
@@ -22,12 +22,12 @@
         ]
     ],
     "checkver": {
-        "url": "https://download.geogebra.org/installers/5.0/version.txt",
+        "url": "https://download.geogebra.org/installers/5.2/version.txt",
         "regex": "(\\d+)-(\\d+)-(\\d+)-(\\d+)",
         "replace": "${1}.${2}.${3}.${4}"
     },
     "autoupdate": {
-        "url": "https://download.geogebra.org/installers/5.0/GeoGebra-Windows-Portable-$dashVersion.zip",
+        "url": "https://download.geogebra.org/installers/$majorVersion.$minorVersion/GeoGebra-Windows-Portable-$dashVersion.zip",
```

```suggestion
        "url": "https://download.geogebra.org/installers/5.2/GeoGebra-Windows-Portable-$dashVersion.zip",
```

`checkver` is pinned to 5.2

---

I was thinking that it would be easier if they decide to bump minor version again as we would only need to change `checkver` but as you want